modules/react/skeleton/lib/parts/skeletonText.tsx

```diff
@@ -1,9 +1,11 @@
 import * as React from 'react';
-import styled from '@emotion/styled';
-import {TEXT_BORDER_RADIUS} from './utils';
+
+import {createComponent, styled, StyledType} from '@workday/canvas-kit-react/common';
 import canvas from '@workday/canvas-kit-react/tokens';
 
-const TextContainer = styled('div')({
+import {TEXT_BORDER_RADIUS} from './utils';
+
+const TextContainer = styled('div')<StyledType>({
   marginBottom: canvas.space.m,
 });
 
```

Comment:
Same below, no longer need to extend from `React.HTMLAttributes<HTMLDivElement>`

Reply:
Addressed.